descriptor.md

```diff
@@ -54,11 +54,11 @@ The following fields contain the primary properties that constitute a Descriptor
 
 - **`artifactType`** *string*
 
-  This OPTIONAL property contains the type of an artifact when the descriptor points to an artifact.
-  This is the value of the config descriptor `mediaType` when the descriptor references an [image manifest](manifest.md).
+  This OPTIONAL property contains the type of an artifact when the descriptor points to an [artifact](artifacts-guidance.md).
+  This property MUST be the same as the `artifactType` of the referenced [manifest](manifest.md), or the `mediaType` of the `config` descriptor if an `artifactType` is not set.
   If defined, the value MUST comply with [RFC 6838][rfc6838], including the [naming requirements in its section 4.2][rfc6838-s4.2], and MAY be registered with [IANA][iana].
 
-Descriptors pointing to [`application/vnd.oci.image.manifest.v1+json`](manifest.md) SHOULD include the extended field `platform`, see [Image Index Property Descriptions](image-index.md#image-index-property-descriptions) for details.
+Descriptors referencing an image (as defined in the [artifacts guidance](artifacts-guidance.md)) SHOULD include the extended field `platform`, see [Image Index Property Descriptions](image-index.md#image-index-property-descriptions) for details. Artifacts as defined in the artifacts guidance MAY include the extended field `platform` when it improves discoverability or interoperability of the artifact. 
```

Comment:
```suggestion
Descriptors referencing an Image (as defined in the [artifacts guidance](artifacts-guidance.md)) SHOULD include the extended field `platform`.
See [Image Index Property Descriptions](image-index.md#image-index-property-descriptions) for details.
Artifacts as defined in the artifacts guidance MAY include the extended field `platform` when it improves discoverability or interoperability of the artifact. 
```

Reply:
I've avoided using the capital/definite Image here (restricting it to artifacts-guidance); do we think it's a good idea to leak the definite article into this file?